frontend/src/utils/address.ts

```diff
@@ -86,18 +87,13 @@ export const toAddress = utils.toAddress;
 // =============================================== Bulk Address Helpers ================================================
 // Duplicates of the above methods for operating on multiple addresses in parallel
 
-export const formatAddresses = (addresses: string[]) => addresses.map(formatAddress);
+export const formatAddresses = (addresses: string[]) => addresses.map(formatNameOrAddress);
 
 export const lookupAddresses = async (addresses: string[], provider: Provider) => {
   const promises = addresses.map((address) => lookupAddress(address, provider));
   return Promise.all(promises);
 };
 
-export const lookupOrFormatAddresses = async (addresses: string[], provider: Provider) => {
-  const promises = addresses.map((address) => lookupOrFormatAddress(address, provider));
```

Comment:
It seems we also don't use `lookupOrFormatAddress` and can delete that too

Reply:
deleted!